configs/apps/all/spack.yaml

```diff
@@ -9,7 +9,6 @@ spack:
     - ${SPACK_STACK_DIR}/configs/repos/repos.yaml
 
   specs:
-    - jedi-fv3-bundle-env
-    - jedi-ufs-bundle-env
-    - ufs-weather-model-bundle-env
+    - jedi-env
+    - ufs-weather-model-env
     - nceplibs-bundle
```

Comment:
@kgerheiser Is nceplibs-bundle redundant, or does it contain ncep libraries that are not built as part of ufs-weather-model-env?

Reply:
It builds things that ufs-weather-model doesn't necessarily use